cmd/thanos/tools.go

```diff
@@ -18,21 +18,30 @@ import (
 	"github.com/thanos-io/thanos/pkg/rules"
 )
 
+type toolsConfig struct {
```

Comment:
It is better to be more specific here. What about `checkRulesConfig`?

Reply:
done in my latest commit!